atlasdb-conjure/src/main/java/com/palantir/atlasdb/http/v2/ClientOptions.java

```diff
@@ -48,6 +48,10 @@
     // Should not be reduced below 65 seconds to support workflows involving locking.
     static final Duration BLOCKING_READ_TIMEOUT = Duration.ofSeconds(65);
 
+    // Time limit for retrying requests which in turn retry 308s that have reached their 20 retry limit defined
+    // in conjure-java-runtime
+    static final Duration REDIRECT_RETRY_TIME_LIMIT = Duration.ofSeconds(15);
```

Comment:
Unclear if this should go here, this feels like it is at a different level of abstraction (the other things here are actually configurations on the CJR client, while this one is not)

Reply:
Given that we found that the retry limit we thought was possibly not the cause, I think we should talk about what we actually want to increase before merging anyway.